### PR TITLE
feat: api ci cd

### LINKED ISSUE #2: Set GitHub Actions of firebase functions CI

### LINKED ISSUE #3: Set GitHub Actions of firebase functions CD

### PR DESCRIPTION
**Pull request by YasunoriMATSUOKA**
close #2 
close #3

### CODE REVIEW

File: .github/workflows/cd-api-firebase-functions.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+name: CD API Firebase Functions
+
+on: [pull_request]
```
Comment (next-web-technology-admin) [inline .github/workflows/cd-api-firebase-functions.yml:3]: @YasunoriMATSUOKA 
CD should not be triggered on pull request.
CD should be triggered only on main pushed.

**Review by next-web-technology-admin — CHANGES_REQUESTED** [review]
@YasunoriMATSUOKA 
API firebase functions CD should be triggerd on pull request.
It should be triggered on main pushed.
Fix it.

File: .github/workflows/cd-api-firebase-functions.yml (hunk 1 above)
Comment (YasunoriMATSUOKA) [reply .github/workflows/cd-api-firebase-functions.yml]: @next-web-technology-admin 
Fixed!

**Review by next-web-technology-admin — APPROVED** [review]
OK

File: .github/workflows/cd-api-firebase-functions.yml (hunk 1 above)
Comment (next-web-technology-admin) [reply .github/workflows/cd-api-firebase-functions.yml]: @YasunoriMATSUOKA 
Confirmed!

**Pull request merged**